### ISSUE TITLE
[Question][nnstreamer] tensorflow support?

### ISSUE THREAD

**Issue by Podossiu** [SKKU-ESLAB/ANT#279]
On the nnstreamer tutorial website, I can see the word that nnstreamer supports the TensorFlow model with the nnstreamer-TensorFlow API. but I can't find that module on ubuntu PPA

Does the nnstreamer doesn't support TensorFlow anymore?

**Comment by asdfrv100** [comment]
yes. you are right. they doesn't support TF anymore.

**Comment by myungjoo** [comment]
NO. we support TF. We just don't build Tensorflow in PPA (tensorflow build requires too much storage and Ubuntu doesn't want to give us the required stroage space).

You can build nnstreamer-tensorflow in your own Ubuntu machine.

Again, nnstreamer DOES support tensorflow.

**Comment by LeeHayun** [comment]
This issue has been inactive for a long time and no longer reflects our current priorities. I'm closing it for now. If you believe this matter should be revisited or if you have new insights, please feel free to open a new issue so we can take a fresh look. Thank you for your contributions.
